src/Behat/Config/Suite.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+final class Suite
+{
+    public function __construct(
+        private string $name,
+        private array $settings = [],
+        private string $profile = 'default',
```

Comment:
Do we still want to keep the option of setting the profile name when creating the Suite? I don't think we need this now that we have the `withSuite` function in the Profile class

Reply:
Oh... I've just forgot about it. thx

Reply:
done